src/lib/RoutingManager.ts

```diff
@@ -70,9 +70,9 @@ class RoutingManager {
     this.router.write(route);
   }
 
-  public dispose({ helper, state }): void {
+  public dispose(): void {
     if (this.router.dispose) {
-      this.router.dispose({ helper, state });
```

Comment:
we need to mention this in the migration guide

Reply:
https://github.com/algolia/instantsearch.js/pull/4120/commits/dcb3c72f2c70cca582224b8a49683a0e594c6ef2